assume/common/units_operator.py

```diff
@@ -465,6 +461,9 @@ async def submit_bids(self, opening: OpeningMessage, meta: MetaDict) -> None:
             market_config=market,
             product_tuples=products,
         )
+
+        orderbook = convert_tensors(orderbook)
```

Comment:
This should only be done for learning strategies somehow - otherwise we have different behavior depending on having torch installed, which is weird.

This fails the integration test, as the dataframes here are different than expected.

We would be back to the situation that we would need `UnitsOperatorDirectStrategy` normally, but as soon as one learning strategy is added, we need `UnitsOperatorDirectStrategyWithConvert` - which is not ideal either.

Best would be to adjust the convert_tensors function to pass the test - aka not change the format if no tensors are there.